chemprop/args.py

```diff
@@ -265,6 +265,8 @@ class TrainArgs(CommonArgs):
     """Additional metrics to use to evaluate the model. Not used for early stopping."""
     save_dir: str = None
     """Directory where model checkpoints will be saved."""
+    checkpoint_frzn: str = None
+    """Directory where model checkpoints be loaded from for overwriting and freezing weights."""
```

Comment:
Slight language change to specify that it's one checkpoint file and the path is to the file and not the directory

"Path to model checkpoint file to be loaded for overwriting and freezing weights."